client/app/components/navbar/navbar.service.js

```diff
@@ -81,15 +80,17 @@ export default class Navbar {
       stateParams: {
         productId: domain.name,
       },
-      subLinks: _.map(domain.subProducts, subDomain => ({
-        name: subDomain.name,
-        title: subDomain.displayName,
-        state: 'app.domain.alldom',
-        stateParams: {
-          allDom: domain.displayName,
-          productId: subDomain.name,
-        },
-      })),
+      subLinks: domain.subProducts
```

Comment:
Weird, I thought `Lodash`'s `map` was immune to this kind of errors

Reply:
I will take a closer look to `<oui-navbar-menu>` because without doing that changes, I'm end up with an empty domains' list.